analyzer/build.gradle.kts

```diff
@@ -36,4 +37,6 @@ dependencies {
     implementation("org.apache.maven.resolver:maven-resolver-transport-wagon:$mavenResolverVersion")
 
     implementation("org.gradle:gradle-tooling-api:${gradle.gradleVersion}")
+
```

Comment:
Nit: I don't think we need a blank line here.

Reply:
Done.